api/HTMLScriptElement.json

```diff
@@ -777,6 +871,100 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "accepts_TrustedScript": {
+          "__compat": {
+            "description": "Can be set with a `TrustedScript`",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLScriptElement/text",
+            "spec_url": "https://w3c.github.io/trusted-types/dist/spec/#dom-htmlscriptelement-text",
+            "tags": [
+              "web-features:script",
+              "web-features:trusted-types"
+            ],
+            "support": {
+              "chrome": {
+                "version_added": "83"
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "135",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "dom.security.trusted_types.enabled",
+                    "value_to_set": "true"
+                  }
+                ]
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": "mirror",
+              "samsunginternet_android": "mirror",
+              "webview_android": "mirror",
+              "webview_ios": "mirror"
+            },
+            "status": {
+              "experimental": true,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        }
+      },
+      "textContent": {
```

Comment:
Same issue as for innerText (as you noted).

Reply:
Done in https://github.com/mdn/browser-compat-data/pull/25839/commits/285119341a024a399e776f6f6829146f7740ac65